### PR TITLE
Add Istio authorization policy for finance portal

### PR DESCRIPTION
**Pull request by kalinkrustev**

### CODE REVIEW

File: terraform/gitops/generate-files/templates/mojaloop/istio-config.yaml.tpl [hunk 1]
```diff
@@ -435,4 +435,24 @@ spec:
...
+metadata:
+  name: finance-portal-auth
+  namespace: ${istio_external_gateway_namespace}
+spec:
+  selector:
+    matchLabels:
+      app: ${istio_external_gateway_name}
+  action: CUSTOM
+  provider:
+    name: ${oathkeeper_auth_provider_name}
```
Comment (vijayg10) [inline terraform/gitops/generate-files/templates/mojaloop/istio-config.yaml.tpl:451]: You may need to add "oathkeeper_auth_provider_name" to mojaloop.tf file.

File: terraform/gitops/generate-files/templates/mojaloop/istio-config.yaml.tpl [hunk 2]
```diff
@@ -435,4 +435,24 @@ spec:
...
+  rules:
+    - when:
+        - operation:
+            paths:
+              - /api/*
+        - key: connection.sni
+          values: ["${finance_portal_fqdn}", "${finance_portal_fqdn}:*"]
```
Comment (vijayg10) [inline terraform/gitops/generate-files/templates/mojaloop/istio-config.yaml.tpl:458]: And its "portal_fqdn" not "finance_portal_fqdn" from the tf file

File: terraform/gitops/generate-files/templates/mojaloop/istio-config.yaml.tpl (hunk 1 above)
Comment (dfry) [reply terraform/gitops/generate-files/templates/mojaloop/istio-config.yaml.tpl:451]: oathkeeper_auth_provider_name  is already there.
@kalinkrustev, you want to move this to PR 75?

Comment (vijayg10) [reply terraform/gitops/generate-files/templates/mojaloop/istio-config.yaml.tpl:451]: Ok

**Comment by kalinkrustev** [comment]
moved to #75

**Pull request closed without merging**